pkg/extensions/sync/sync_test.go

```diff
@@ -3461,6 +3531,62 @@ func TestSignatures(t *testing.T) {
 		So(err, ShouldBeNil)
 		So(resp, ShouldNotBeEmpty)
 		So(resp.StatusCode(), ShouldEqual, http.StatusOK)
+		So(godigest.FromBytes(resp.Body()), ShouldEqual, sbomDigest)
+
+		// verify sbom signature
+		sbom := fmt.Sprintf("localhost:%s/%s@%s", destPort, repoName, sbomDigest)
+
+		test.LoadNotationPath(tdir)
+		// notation verify signed sbom
+		err = test.VerifyWithNotation(sbom, tdir)
+		So(err, ShouldBeNil)
+
+		// a = &options.AnnotationOptions{Annotations: []string{fmt.Sprintf("tag=%s", sbomCosignTag)}}
```

Comment:
Why are the annotation options removed?
BTW this commented code needs to be updated.

Reply:
ok